src/components/GlobalErrorBoundary/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Card, Text, Title, Button } from '@gnosis.pm/safe-react-components'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+type Props = {
+  error: Error
+  componentStack: string
+  resetError: () => void
+}
+
+const GlobalErrorBoundaryFallback: React.FC<Props> = ({ error, componentStack }: Props) => {
+  return (
+    <Wrapper>
+      <Card>
+        <Title size="md">There was an error</Title>
+        {process.env.NODE_ENV !== 'production' && (
+          <>
+            <Text size="xl" color="error">
+              {error.toString()}
+            </Text>
+            <Text size="md" color="error">
+              {componentStack}
+            </Text>
+          </>
+        )}
+        <Button
+          size="md"
+          color="primary"
+          onClick={() => {
+            window.location.href = '/app/'
+          }}
+        >
+          Go to Landing
+        </Button>
```

Comment:
This is a link

Reply:
right, we should add a `Link` component in Components.

Reply:
@mikheevm do you know if we already have a link with the styles of a button in Gnosis? otherwise, I would leave it as it is and create a ticket in Components

Reply:
Ticket created https://github.com/gnosis/safe-react-components/issues/72.

Reply:
@mikheevm I just started the ticket to add the Link component, but, should we use links as buttons? Technically we could, we only need to "copy" all the material-ui styles for the button and apply them to the anchor element. But does it make sense? shouldn't we use the links with a different style than buttons? they are used for different actions so it's reasonable they should use different styles. 

What I'm suggesting is that we should change the button and use the link style, like in here: 
![image](https://user-images.githubusercontent.com/4376509/97577262-f38c9980-19cd-11eb-8cc8-cd8c2c412fa4.png)

Reply:
cc: @tschubotz @fernandomg @dasanra @Agupane

Reply:
TLTR, I think we should separate the actions from navigation, meaning, if we want to move inside the page or to another page, we should use links with link styles. If we want to trigger an action, we should use buttons with button styles. 

Giving a quick review of the app the only inconsistency I found was in the nav of each section:
![image](https://user-images.githubusercontent.com/4376509/97581457-5b91ae80-19d3-11eb-8c71-f2c720d9865a.png)
It should have the button style.